pkg/alert/rule_controller.go

```diff
@@ -58,6 +58,8 @@ type RuleController struct {
 }
 
 // NewRuleController returns a new AlertingRule controller instance.
+//
+//nolint:errcheck
```

Comment:
missed this one in my previous review but here also I'd prefer to check for errors if possible

Reply:
missed it, done!